monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -115,7 +115,7 @@ class AppComponent extends AuthComponent {
   setMode = () => {
       return IslandHttpClient.get('/api/island/mode')
       .then(res => {
-        this.setState({islandMode: res.body.mode});
+        this.setState({islandMode: res.body});
```

Comment:
Might need `res.json()`, not sure.

Reply:
`res.body` works
![image](https://user-images.githubusercontent.com/44770317/182357983-7d23744f-8015-4661-afdd-21e6f17332bd.png)